<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="InvalidDataException.xml" source-language="en-US" target-language="it-IT">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5603ce5cade9b77b52ce6f41c81bef9648bd1234b.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">603ce5cade9b77b52ce6f41c81bef9648bd1234b</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netcore-1.0,netcore-1.1,netcore-2.0,netcore-2.1,netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7,netstandard-1.0,netstandard-1.1,netstandard-1.2,netstandard-1.3,netstandard-1.4,netstandard-1.5,netstandard-1.6,netstandard-2.0,xamarinandroid-7.1,xamarinios-10.8,xamarinmac-3.0</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.IO.InvalidDataException">
          <source>The exception that is thrown when a data stream is in an invalid format.</source>
          <target state="translated">Eccezione generata quando un flusso di dati è in un formato non valido.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.InvalidDataException">
          <source>An <ph id="ph1">&lt;xref:System.IO.InvalidDataException&gt;</ph> is thrown when invalid data is detected in the data stream.</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.IO.InvalidDataException&gt;</ph> viene generata quando vengono rilevati dati non validi nel flusso di dati.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:System.IO.InvalidDataException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.InvalidDataException">
          <source>An <ph id="ph1">&lt;xref:System.IO.InvalidDataException&gt;</ph> is thrown when invalid data is detected in the data stream, such as in the following situations:</source>
          <target state="translated">Un <ph id="ph1">&lt;xref:System.IO.InvalidDataException&gt;</ph> viene generata quando i dati non validi viene rilevati nel flusso di dati, ad esempio nelle situazioni seguenti:</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.InvalidDataException">
          <source>A row of data in a delimited text file does not have the expected number of fields.</source>
          <target state="translated">Una riga di dati in un file di testo delimitato non contiene il numero previsto di campi.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.InvalidDataException">
          <source>An attempt is made to read a binary file, such as word processing document, that has become corrupted.</source>
          <target state="translated">Viene effettuato un tentativo di leggere un file binario, ad esempio un documento di elaborazione testi, che è stato danneggiato.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.IO.InvalidDataException">
          <source>An attempt is made to read bytes from a decompression stream that has an invalid format.</source>
          <target state="translated">Viene effettuato un tentativo di leggere i byte da un flusso di decompressione che ha un formato non valido.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> class.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "An invalid argument was specified."</source>
          <target state="translated">Questo costruttore inizializza la <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l'errore, ad esempio "è stato specificato un argomento non valido".</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor">
          <source>This message is localized based on the current system culture.</source>
          <target state="translated">Questo messaggio è localizzato in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> con un messaggio di errore specificato.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance to a system-supplied message that describes the error, such as "An invalid argument was specified."</source>
          <target state="translated">Questo costruttore inizializza la <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> proprietà della nuova istanza di un messaggio fornito dal sistema che descrive l'errore, ad esempio "è stato specificato un argomento non valido".</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String)">
          <source>This message is localized based on the current system culture.</source>
          <target state="translated">Questo messaggio è localizzato in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Messaggio di errore in cui viene indicato il motivo dell'eccezione.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">Eccezione che ha determinato l'eccezione corrente.</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated">Se il parametro <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> non è <ph id="ph1">&lt;see langword="null" /&gt;</ph>, l'eccezione corrente viene generata in un blocco <ph id="ph2">&lt;see langword="catch" /&gt;</ph> in cui viene gestita l'eccezione interna.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> class with a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Inizializza una nuova istanza della classe <ph id="ph1">&lt;see cref="T:System.IO.InvalidDataException" /&gt;</ph> con un riferimento all'eccezione interna che è la causa dell'eccezione.</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>This constructor initializes the <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> property of the new instance using the value of the <ph id="ph2">`message`</ph> parameter.</source>
          <target state="translated">Questo costruttore inizializza la <ph id="ph1">&lt;xref:System.Exception.Message%2A&gt;</ph> proprietà della nuova istanza utilizzando il valore della <ph id="ph2">`message`</ph> parametro.</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>The content of the <ph id="ph1">`message`</ph> parameter is intended to be understood by humans.</source>
          <target state="translated">Il contenuto del parametro <ph id="ph1">`message`</ph> deve essere facilmente comprensibile.</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>The caller of this constructor is required to ensure that this string has been localized for the current system culture.</source>
          <target state="translated">Il chiamante del costruttore deve garantire che la stringa sia stata localizzata in base alle impostazioni cultura correnti del sistema.</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property.</source>
          <target state="translated">Un'eccezione generata come risultato diretto di un'eccezione precedente deve includere un riferimento all'eccezione precedente nella proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph>.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.IO.InvalidDataException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property returns the same value that is passed into the constructor, or <ph id="ph2">`null`</ph> if the <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> property does not supply the inner exception value to the constructor.</source>
          <target state="translated">La proprietà <ph id="ph1">&lt;xref:System.Exception.InnerException%2A&gt;</ph> restituisce lo stesso valore passato al costruttore oppure <ph id="ph2">`null`</ph> nel caso in cui la proprietà <ph id="ph3">&lt;xref:System.Exception.InnerException%2A&gt;</ph> non fornisca il valore di eccezione interna al costruttore.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>